make/bundle-dist

```diff
@@ -19,20 +19,23 @@ else
     echo "Unkown stemcell operating system: ${FISSILE_STEMCELL}"
     exit 1
 fi
-ARCHIVE="${GIT_ROOT}/output/scf-${stemcell_os}-${APP_VERSION}.zip"
+ARCHIVE_ROOT="${GIT_ROOT}/output/scf-${stemcell_os}-${APP_VERSION}"
 
-echo Packaging, taking $APP_VERSION ...
+echo Packaging, taking "${APP_VERSION}" ...
 
 # Assembling the pieces ...
 mkdir -p ${tmp_dir}/kube ${tmp_dir}/helm
 
 # kube configs
-unzip ${GIT_ROOT}/output/${ARTIFACT_NAME}-kube-${APP_VERSION}.zip -d ${tmp_dir}/kube
+tar xf "${GIT_ROOT}/output/${ARTIFACT_NAME}-kube-${APP_VERSION}.tgz" -C "${tmp_dir}/kube"
 # helm charts
-unzip ${GIT_ROOT}/output/${ARTIFACT_NAME}-helm-${APP_VERSION}.zip -d ${tmp_dir}/helm
+tar xf "${GIT_ROOT}/output/${ARTIFACT_NAME}-helm-${APP_VERSION}.tgz" -C "${tmp_dir}/helm"
 
 # "Am I Ok" for kube
 cp ${GIT_ROOT}/bin/dev/kube-ready-state-check.sh ${tmp_dir}/
 
 # Package the assembly. This directly places it into the output
-( cd ${tmp_dir} ; zip -r9 $ARCHIVE * )
+# For now, include both zip and tgz in order to transition smoothly
+cd "${tmp_dir}"
+tar czvf "${ARCHIVE_ROOT}.tgz" -- *
+zip -r9 "${ARCHIVE_ROOT}.zip" -- *
```

Comment:
Doesn't this include the `*.tgz` file inside the `*.zip`?

Reply:
Oops, I forgot that `ARCHIVE_ROOT` includes the full path and is outside the temp directory.